src/main/java/database/Storage.java

```diff
@@ -0,0 +1,16 @@
+package database;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> assortment = new HashMap<>();
+
+    public static Map<String, Integer> getAssortment() {
+        return Map.copyOf(assortment);
+    }
+
+    public static void updateDataBase(Map<String, Integer> transactions) {
```

Comment:
This method allows to update the Storage with a map of transactions. However, this is not a safe operation, because it allows to overwrite existing entries in the `assortment` map. I would suggest to provide more specific methods for updating the storage, like `addFruit(Fruit fruit, int quantity)` and `removeFruit(Fruit fruit, int quantity)`. These methods should also check if the operation is valid (for example, you can't remove more fruits than there are in the storage). If the operation is not valid, they should throw an exception.